eslint.config.mjs

```diff
@@ -895,6 +884,20 @@ export default typescript.config([
       ...strictRules,
     },
   },
+  {
+    name: '@emotion',
+    plugins: {
+      '@emotion': emotion,
+    },
+    rules: {
+      '@emotion/import-from-emotion': 'error',
+      '@emotion/jsx-import': 'off',
+      '@emotion/no-vanilla': 'error',
+      '@emotion/pkg-renaming': 'error',
+      '@emotion/styled-import': 'error',
+      '@emotion/syntax-preference': ['off', 'string'], // TODO(ryan953): Enable this so `css={css``}` is required
```

Comment:
This last one is a new one on the list, even though it's disabled.

I think we should enable it because it'll warn against places that use `css={}` without `import {css} from '@emotion/react';` passed in. 
It also requires that we go all-in on string syntax over object syntax (as is the current preference)

If it were enabled we'd be doing this all the time:
```javascript
styled('div')`
  background: red;
`;
```
and never:
```javascript
styled('div')({
  background: 'red',
});
```